sql/item_mods.sql

```diff
@@ -20093,6 +20093,11 @@ INSERT INTO `item_mods` VALUES (16550,431,1);
 INSERT INTO `item_mods` VALUES (16557,23,6);
 INSERT INTO `item_mods` VALUES (16557,25,3);
 INSERT INTO `item_mods` VALUES (16562,9,1);
+INSERT INTO `item_mods` VALUES (16564,431,1); -- Flame Blade: AddEffect Type NORMAL
```

Comment:
On SQL in general, I assume that the item mods for the deleted effect scripts still need to be added?

Reply:
yes. I have an open issue for it, and before the branch is ready a lot of sql work will need done